SPECS/pcsc-lite-ccid/pcsc-lite-ccid.spec

```diff
@@ -0,0 +1,342 @@
+%global dropdir %(pkg-config libpcsclite --variable usbdropdir 2>/dev/null)
+%global pcsc_lite_ver 1.8.9
+Summary:        Generic USB CCID smart card reader driver
+Name:           pcsc-lite-ccid
+Version:        1.4.33
+Release:        2%{dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://ccid.apdu.fr/files
+Source0:        https://ccid.apdu.fr/files/ccid-%{version}.tar.bz2
+Source1:        https://ccid.apdu.fr/files/ccid-%{version}.tar.bz2.asc
+Source2:        gpgkey-F5E11B9FFE911146F41D953D78A1B4DFE8F9C57E.gpg
```

Comment:
Our sources are using the hash signatures already, so we can remove these two sources, their signatures, and the line where they are used.